averages.js

```diff
@@ -0,0 +1,45 @@
+function total(array){
+  return array.reduce((a,b) => a+b, 0)
+};
+
+function mean(array){
+  return total(array)/array.length
+};
+
+function median(array){
```

Comment:
Remember that to find the median we would want to sort the array first before grabbing the middle value(s)